modules/storage-persistent-storage-volume-encrypt-with-kms-key.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage-aws.adoc
+
+[id="EBS-volume-encrypt_{context}"]
+= Encrypting EBS instance volumes with a KMS key
+
+.Prerequisites
+
+* Storage must exist in the underlying infrastructure before you encrypt the EBS instance volumes in {product-title}.
+
+.Procedure
+
+Defining a KMS key to encrypt EBS instance volumes is useful when you have explicit compliance and security guidelines when deploying to AWS.
+
+. Create a storage class:
++
+[source,yaml]
+----
+$ cat << EOF | oc create -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+name: <storage-class-name> <1>
+parameters:
+fsType: ext4 <2>
+kmsKeyId: keyvalue <3>
+provisioner: ebs.csi.aws.com
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer
+EOF
+----
+<1> Specifies the name of the storage class.
+<2> File system that is created on provisioned volumes.
+<3> The full Amazon Resource Name (ARN) of the key to use when encrypting the volume. If none is supplied, but encypted is set to true, then AWS generates a key. See the link:https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html[AWS documentation] for a valid ARN value.
```

Comment:
The parameter should be kms key ID and it could get from the aws offical doc: https://docs.aws.amazon.com/kms/latest/developerguide/find-cmk-id-arn.html

Reply:
Addressed